src/main/java/com/example/demo/controller/FileController.java

```diff
@@ -0,0 +1,69 @@
+package com.example.demo.controller;
+
+import com.example.demo.entity.FileInformation;
+import com.example.demo.service.FileService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import org.springframework.core.io.Resource;
+
+@RestController
+@RequestMapping("/file")
+public class FileController {
+
+    FileService fileService;
+
+    @Autowired
+    public FileController(FileService fileService) {
```

Comment:
Use lombok to create a constructor. 
Have a look at `RequireArgsConstructor` annotation